dev-requirements.txt

```diff
@@ -11,3 +11,6 @@ bandit~=1.6
 
 # For commit message linting in circleci we need GitPython
 GitPython~=2.1
+
+# PyYAML builds from source and needs bdist_wheel
```

Comment:
I tested this out and it doesn't work. Unfortunately, pip will only install packages listed in alphabetical order. So this change will not work. I will submit a PR with a doc change in the development setup.